internal/oci/uvm.go

```diff
@@ -116,6 +119,8 @@ const (
 	annotationBootFilesRootPath          = "io.microsoft.virtualmachine.lcow.bootfilesrootpath"
 	annotationStorageQoSBandwidthMaximum = "io.microsoft.virtualmachine.storageqos.bandwidthmaximum"
 	annotationStorageQoSIopsMaximum      = "io.microsoft.virtualmachine.storageqos.iopsmaximum"
+	annotationKernelDirectBoot           = "io.microsoft.virtualmachine.kerneldirectboot"
```

Comment:
I assume you adding this means there wasn't a way via CRI to disable kernel direct boot before?

Reply:
Not that I know of. When we create a(n?) uvm we only checked the build number of the host to determine whether or not to use direct boot.